### DISCUSSION TITLE
Sudo yarn build fails sometimes

### DISCUSSION THREAD

**Discussion by finch986** [polonel/trudesk#641] [Q&A]
OS : Ubuntu , Whenever I am doing yarn build on remote server and connection breaks . Then I login again and do yarn build it says 

sudo yarn build --force
Running "uglify:uikit" (uglify) task
>> 1 file created 318 kB → 199 kB

Running "cssmin:target" (cssmin) task
>> 1 file created. 506 kB → 384 kB

Running "cssmin:minify" (cssmin) task
>> 3 files created. 392 kB → 386 kB

Running "shell:webpackDist" (shell) task
No matches found: "public/js/*"
Warning: Command failed: yarn run webpackdist
No matches found: "public/js/*"
 Used --force, continuing.

Done, but with warnings.

js folder is empty .. Please help

**Comment by polonel** [comment]
This would be an issue with your connection and not the install script. 

When you lose connection the script is still running. Then you try to execute the script again while it is still running.

The best approach is to kill the existing running install script before running again.

**Reply by finch986** [reply]
It deletes all the files in public/js folder . I have alredy tried killing the currently running excuting script . Its not working
